.eslintrc

```diff
@@ -25,7 +25,6 @@
         "no-else-return": 2,
         "no-empty": 2,
         "no-empty-character-class": 2,
-        "no-empty-label": 2,
```

Comment:
This looks like it was replaced by http://eslint.org/docs/rules/no-unused-labels